### PR TITLE
Document SizeMemoryBackedVolumes feature for 1.20

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Related KEP: https://github.com/kubernetes/enhancements/pull/1968

### CODE REVIEW

**Comment by annajung** [comment]
/milestone 1.20
/hold no k/k pr
/assign
/sig node

enhancement: https://github.com/kubernetes/enhancements/issues/1967

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -303,6 +303,12 @@ While tmpfs is very fast, be aware that unlike disks, tmpfs is cleared on
 node reboot and any files you write count against your container's
 memory limit.
 
+{{< note >}}
+If the `SizeMemoryBackedVolumes` feature gate is enabled in 1.20+, users may
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md:307]: I'd write this:
```suggestion
If the `SizeMemoryBackedVolumes` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled, you can
```

Comment (sftim) [reply content/en/docs/concepts/storage/volumes.md:307]: Remember to update `/docs/reference/command-line-tools-reference/feature-gates/`

**Comment by annajung** [comment]
/hold cancel
k/k pr merged

Hi @derekwaynecarr w/ k/k pr merged in, Sig Docs would appreciate it if you could fill the content of the docs early, ideally before kubecon to get it reviewed/merged on time. Thank you!

**Comment by sftim** [comment]
Hi @derekwaynecarr 

Would you have time to make sure that `/docs/reference/command-line-tools-reference/feature-gates/` is up to date for the changes this PR is documenting?

**Comment by annajung** [comment]
Hi @derekwaynecarr today is the deadline for docs to be in ready for review state. Please address Tim's comment above and let us know if this is ready to be reviewed.

**Comment by derekwaynecarr** [comment]
I made the recommended update, and believe this is ready for review.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -148,6 +148,7 @@ different Kubernetes components.
 | `ServiceNodeExclusion` | `false` | Alpha | 1.8 | 1.18 |
 | `ServiceNodeExclusion` | `true` | Beta | 1.19 | |
 | `ServiceTopology` | `false` | Alpha | 1.17 | |
+| `SizeMemoryBackedVolumes` | `false` | Alpha | 1.20 | |
```
Comment (annajung) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:151]: Just need a short description below describing this feature. Then, I think docs are good to merge

**Review by sftim — COMMENTED** [review]
I wonder if https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/ should also get a mention of the new feature gate. What do you think @derekwaynecarr ?

**Comment by sftim** [comment]
Hi @derekwaynecarr 

Repeating https://github.com/kubernetes/website/pull/24906#discussion_r529001457 - we'd like a summary of the new feature gate, similar to the other ones already in the list.

**Comment by reylejano** [comment]
Hi @derekwaynecarr, just a reminder that the docs merge deadline is in a few days on Dec 2. If you can address @sftim 's comment about adding a summary of the new feature gate then this PR can merge after a quick review. Thank you

**Comment by derekwaynecarr** [comment]
@annajung addressed feedback

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -527,6 +528,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `ServiceNodeExclusion`: Enable the exclusion of nodes from load balancers created by a cloud provider.
   A node is eligible for exclusion if labelled with "`alpha.service-controller.kubernetes.io/exclude-balancer`" key or `node.kubernetes.io/exclude-from-external-load-balancers`.
 - `ServiceTopology`: Enable service to route traffic based upon the Node topology of the cluster. See [ServiceTopology](/docs/concepts/services-networking/service-topology/) for more details.
+- `SizeMemoryBackedVolumes`: Enables kubelet support to size memory backed volumes.  See [volumes] (docs/concepts/storage/volumes) for more details.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:531]: CommonMark doesn't like the extra space
```suggestion
- `SizeMemoryBackedVolumes`: Enables kubelet support to size memory backed volumes.  See [volumes](docs/concepts/storage/volumes) for more details.
```

**Review by sftim — CHANGES_REQUESTED** [review]
One fix needed; otherwise LGTM.

**Comment by derekwaynecarr** [comment]
@sftim fixed typo.

**Pull request merged**